src/comments.js

```diff
@@ -933,6 +938,38 @@ function printComment(commentPath, options) {
   }
 }
 
+function isJsDocComment(comment) {
+  const lines = comment.value.split("\n");
+  return (
+    lines.length > 1 &&
+    lines.slice(0, lines.length - 1).every(line => line.trim()[0] === "*")
+  );
+}
+
+function printJsDocComment(comment, options) {
+  const lines = comment.value.split("\n");
+  const tabWidth = options.tabWidth;
+  const indentSize = util.getIndentSize(comment.value, tabWidth);
+
+  const parts = [];
+
+  lines.forEach((line, index) => {
+    parts.push(
+      docBuilders.align(
+        indentSize,
+        // We only want to start with a leading space if we are not on
+        // the first line of a JSDoc comment.
+        (index > 0 ? " " : "") + line.trimLeft()
```

Comment:
Don’t we want to trim on the right side too? (`.trim()` instead of `.trimLeft()`)

Reply:
I agree.

It does break one of the tests where the `*/` is on the same line as the last line of the comment:

```js
/** This is a regression test for a bug where we forgot to mark the fields of
  * react classes as initialized, when the class was created with createClass().
  * This would manifest as complaining that metric requires an annotation */
```
becomes

```js
/** This is a regression test for a bug where we forgot to mark the fields of
  * react classes as initialized, when the class was created with createClass().
  * This would manifest as complaining that metric requires an annotation*/
```

Do we need to build in an edge-case here? (i.e., `.trimLeft()` if the `*/` is not on its own line?